ospd_openvas/nvticache.py

```diff
@@ -57,12 +58,16 @@ def get_feed_version(self):
 
     def get_oids(self):
         """ Get the list of NVT OIDs.
+        Return a list of lists. Each single list contains the filename
```

Comment:
```suggestion
        Returns:
            list: A list of lists. Each single list contains the filename
```

Reply:
I would prefer to use the google style (http://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google) for return values too